docs/web3-eth-miner.rst

```diff
@@ -0,0 +1,224 @@
+.. _eth-miner:
+
+.. include:: include_announcement.rst
+
+==============
+web3.eth.miner
```

Comment:
I would change this title to: ``Web3 Miner Module``

Reply:
So, this will be web3.miner. right?

Reply:
The Title should be ``Web3 Miner Module`` and method examples should be without the web3 umbrella package e.g.: ``miner.setExtra()`` because the miner module shouldn't be in the standard eth module.